### ISSUE TITLE
dlls not working

### ISSUE THREAD

**Issue by 14cky** [EugeneSunrise/reWASD#10]
yea, i know, that there's another issue thread. But it's not working. After installing everything, replacing dlls, but after importing config, creating a new group, grouping keyboard with mouse, pressing "Apply to Slot 1", nothing happens

**Comment by bravazeroh** [comment]
It was working fine for me yesterday, I think it was because I still had free trial remaining, however now that it ran out it's not working anymore, can't apply a config.

**Comment by bravazeroh** [comment]
Nevermind, I reinstalled and it's working again. When I followed the installers instructions I didn't restart. I replaced the 2 DLLs first in the main folder, then I restarted and now it seems to be working. (I also unblocked the DLL files from their properties)

**Comment by EugeneSunrise** [comment]
wait

**Comment by Protoup20** [comment]
> Nevermind, I reinstalled and it's working again. When I followed the installers instructions I didn't restart. I replaced the 2 DLLs first in the main folder, then I restarted and now it seems to be working. (I also unblocked the DLL files from their properties)

so you can turn the remap on? bc thats the one thing i cant which is the important part

**Comment by bravazeroh** [comment]
> > Nevermind, I reinstalled and it's working again. When I followed the installers instructions I didn't restart. I replaced the 2 DLLs first in the main folder, then I restarted and now it seems to be working. (I also unblocked the DLL files from their properties)
> 
> 
> 
> so you can turn the remap on? bc thats the one thing i cant which is the important part

Yeah I can

**Comment by JugadveerSingh** [comment]
> > > Nevermind, I reinstalled and it's working again. When I followed the installers instructions I didn't restart. I replaced the 2 DLLs first in the main folder, then I restarted and now it seems to be working. (I also unblocked the DLL files from their properties)
> > 
> > 
> > so you can turn the remap on? bc thats the one thing i cant which is the important part
> 
> Yeah I can

i did everything right but the apply to slot 1 button does nothing

**Comment by Protoup20** [comment]
> > > Nevermind, I reinstalled and it's working again. When I followed the installers instructions I didn't restart. I replaced the 2 DLLs first in the main folder, then I restarted and now it seems to be working. (I also unblocked the DLL files from their properties)
> > 
> > 
> > so you can turn the remap on? bc thats the one thing i cant which is the important part
> 
> Yeah I can

did you use the trial version, if so was it expired?
also can you post a screenshot of the remap powerbutton being on

**Comment by DDGAMEDD1** [comment]
when i click apply to slot 1 nothin happens and button stays yellow any fix? i had reWASD before with trial

**Comment by bravazeroh** [comment]
> > > > Nevermind, I reinstalled and it's working again. When I followed the installers instructions I didn't restart. I replaced the 2 DLLs first in the main folder, then I restarted and now it seems to be working. (I also unblocked the DLL files from their properties)
> > > 
> > > 
> > > so you can turn the remap on? bc thats the one thing i cant which is the important part
> > 
> > 
> > Yeah I can
> 
> did you use the trial version, if so was it expired? also can you post a screenshot of the remap powerbutton being on

Yes, everything had expired and here's a video where you can see the license is not normal and also me enabling the cfg, and disabling it multiple times.

https://github.com/EugeneSunrise/reWASD/assets/153247854/daedd9ab-8d16-447b-b67a-1570765c71d9

**Comment by 14cky** [comment]
> Nevermind, I reinstalled and it's working again. When I followed the installers instructions I didn't restart. I replaced the 2 DLLs first in the main folder, then I restarted and now it seems to be working. (I also unblocked the DLL files from their properties)

I did everything the same, but it's still not working for me. Idk

https://github.com/EugeneSunrise/reWASD/assets/113927878/2604b11e-e318-401d-8fc2-52e3b5b4d02c

**Comment by Protoup20** [comment]
> > > > > Nevermind, I reinstalled and it's working again. When I followed the installers instructions I didn't restart. I replaced the 2 DLLs first in the main folder, then I restarted and now it seems to be working. (I also unblocked the DLL files from their properties)
> > > > 
> > > > 
> > > > so you can turn the remap on? bc thats the one thing i cant which is the important part
> > > 
> > > 
> > > Yeah I can
> > 
> > 
> > did you use the trial version, if so was it expired? also can you post a screenshot of the remap powerbutton being on
> 
> Yes, everything had expired and here's a video where you can see the license is not normal and also me enabling the cfg, and disabling it multiple times.
> Desktop.2023.12.09.-.20.26.16.03.mp4

did you use revo uninstaller

also did you get a message after trying to turn it on for the 1st time to reboot

**Comment by bravazeroh** [comment]
> > > > > > Nevermind, I reinstalled and it's working again. When I followed the installers instructions I didn't restart. I replaced the 2 DLLs first in the main folder, then I restarted and now it seems to be working. (I also unblocked the DLL files from their properties)
> 
> > > > > 
> 
> > > > > 
> 
> > > > > so you can turn the remap on? bc thats the one thing i cant which is the important part
> 
> > > > 
> 
> > > > 
> 
> > > > Yeah I can
> 
> > > 
> 
> > > 
> 
> > > did you use the trial version, if so was it expired? also can you post a screenshot of the remap powerbutton being on
> 
> > 
> 
> > Yes, everything had expired and here's a video where you can see the license is not normal and also me enabling the cfg, and disabling it multiple times.
> 
> > Desktop.2023.12.09.-.20.26.16.03.mp4
> 
> 
> 
> did you use revo uninstaller
> 
> 
> 
> also did you get a message after trying to turn it on for the 1st time to reboot

No. Literally just windows' own control panel to delete the program and it's files. I don't even know what revo uninstaller is 😄

**Comment by Protoup20** [comment]
> > > > > > > Nevermind, I reinstalled and it's working again. When I followed the installers instructions I didn't restart. I replaced the 2 DLLs first in the main folder, then I restarted and now it seems to be working. (I also unblocked the DLL files from their properties)
> > 
> > 
> > > > > >
> > 
> > 
> > > > > >
> > 
> > 
> > > > > > so you can turn the remap on? bc thats the one thing i cant which is the important part
> > 
> > 
> > > > >
> > 
> > 
> > > > >
> > 
> > 
> > > > > Yeah I can
> > 
> > 
> > > >
> > 
> > 
> > > >
> > 
> > 
> > > > did you use the trial version, if so was it expired? also can you post a screenshot of the remap powerbutton being on
> > 
> > 
> > > 
> > 
> > 
> > > Yes, everything had expired and here's a video where you can see the license is not normal and also me enabling the cfg, and disabling it multiple times.
> > 
> > 
> > > Desktop.2023.12.09.-.20.26.16.03.mp4
> > 
> > 
> > did you use revo uninstaller
> > also did you get a message after trying to turn it on for the 1st time to reboot
> 
> No. Literally just windows' own control panel to delete the program and it's files. I don't even know what revo uninstaller is 😄 I'm also not sure what message you're talking about but the program asks for permission on reboot.

ok let me check if i got the order right, bc i might have done it wrong prev.
you uninstalled the prev version and rebooted
you reinstalled it, BUT did not immedialty reboot. INSTEAD you replaced the dll files first AND THEN REBOOTED, correct?

**Comment by bravazeroh** [comment]
> > > > > > > > Nevermind, I reinstalled and it's working again. When I followed the installers instructions I didn't restart. I replaced the 2 DLLs first in the main folder, then I restarted and now it seems to be working. (I also unblocked the DLL files from their properties)
> 
> > > 
> 
> > > 
> 
> > > > > > >
> 
> > > 
> 
> > > 
> 
> > > > > > >
> 
> > > 
> 
> > > 
> 
> > > > > > > so you can turn the remap on? bc thats the one thing i cant which is the important part
> 
> > > 
> 
> > > 
> 
> > > > > >
> 
> > > 
> 
> > > 
> 
> > > > > >
> 
> > > 
> 
> > > 
> 
> > > > > > Yeah I can
> 
> > > 
> 
> > > 
> 
> > > > >
> 
> > > 
> 
> > > 
> 
> > > > >
> 
> > > 
> 
> > > 
> 
> > > > > did you use the trial version, if so was it expired? also can you post a screenshot of the remap powerbutton being on
> 
> > > 
> 
> > > 
> 
> > > > 
> 
> > > 
> 
> > > 
> 
> > > > Yes, everything had expired and here's a video where you can see the license is not normal and also me enabling the cfg, and disabling it multiple times.
> 
> > > 
> 
> > > 
> 
> > > > Desktop.2023.12.09.-.20.26.16.03.mp4
> 
> > > 
> 
> > > 
> 
> > > did you use revo uninstaller
> 
> > > also did you get a message after trying to turn it on for the 1st time to reboot
> 
> > 
> 
> > No. Literally just windows' own control panel to delete the program and it's files. I don't even know what revo uninstaller is 😄 I'm also not sure what message you're talking about but the program asks for permission on reboot.
> 
> 
> 
> ok let me check if i got the order right, bc i might have done it wrong prev.
> 
> you uninstalled the prev version and rebooted
> 
> you reinstalled it, BUT did not immedialty reboot. INSTEAD you replaced the dll files first AND THEN REBOOTED, correct?

Yes

**Comment by JJohanXz** [comment]
@Protoup20 Were you lucky enough to work?

**Comment by JJohanXz** [comment]
@Protoup20 Were you lucky enough to work?

**Comment by Protoup20** [comment]
> @Protoup20 Were you lucky enough to work?
nope

**Comment by JugadveerSingh** [comment]
> Nevermind, I reinstalled and it's working again. When I followed the installers instructions I didn't restart. I replaced the 2 DLLs first in the main folder, then I restarted and now it seems to be working. (I also unblocked the DLL files from their properties)

When you turn on reWASD do you get the popup that your free trail has expired?

**Comment by 14cky** [comment]
Nah, it doesnt even work for me on windows 11, do you even need reWASDCommon.dll file?

**Comment by 14cky** [comment]
found a little interesting thing, if you use reWASD junior, after you press apply, it gives you "checking license error"

**Comment by scanlan897** [comment]
icant apply cfg + magic is off

**Comment by albirizo** [comment]
https://github.com/EugeneSunrise/reWASD/assets/129513553/f91c080b-82e7-406c-ad2c-4ce79d6052f4


I've installed it all correctly, no free trial popup; yet I still get this error? how can I fix this?

**Comment by 14cky** [comment]
guys, one way out of this, uninstall rewasd and change SID. Just do that every 3 days, while we wait for fix

**Comment by 14cky** [comment]
> > guys, one way out of this, uninstall rewasd and change SID. Just do that every 3 days, while we wait for fix
> 
> How do you do change SID?

https://www.youtube.com/watch?v=JtLA7sb5Pao&ab_channel=Mysteri0us
but it will log out of your accounts in some apps, like chrome or discord

**Comment by MylesXD** [comment]
> > > guys, one way out of this, uninstall rewasd and change SID. Just do that every 3 days, while we wait for fix
> > 
> > 
> > How do you do change SID?
> 
> https://www.youtube.com/watch?v=JtLA7sb5Pao&ab_channel=Mysteri0us but it will log out of your accounts in some apps, like chrome or discord

cant changing your sid corrupt your data?

**Comment by albirizo** [comment]
> > > > guys, one way out of this, uninstall rewasd and change SID. Just do that every 3 days, while we wait for fix
> > > 
> > > 
> > > How do you do change SID?
> > 
> > 
> > https://www.youtube.com/watch?v=JtLA7sb5Pao&ab_channel=Mysteri0us but it will log out of your accounts in some apps, like chrome or discord
> 
> cant changing your sid corrupt your data?

yep, not doing this definitely.

**Comment by 14cky** [comment]
> > > > guys, one way out of this, uninstall rewasd and change SID. Just do that every 3 days, while we wait for fix
> > > 
> > > 
> > > How do you do change SID?
> > 
> > 
> > https://www.youtube.com/watch?v=JtLA7sb5Pao&ab_channel=Mysteri0us but it will log out of your accounts in some apps, like chrome or discord
> 
> cant changing your sid corrupt your data?

it didn't corrupt mine, idk really

**Comment by EugeneSunrise** [comment]
Wait fix

**Comment by EugeneSunrise** [comment]
CHECK README AND GIVE ME FEEDBACK

**Comment by EugeneSunrise** [comment]
Check releases and give me feedback

**Comment by EugeneSunrise** [comment]
6.6 ver ready, 7.0 will be later, any questions? Create NEW issue
